packages/console/src/hooks/use-connector-groups.ts

```diff
@@ -1,4 +1,5 @@
-import { ConnectorDto, ConnectorType } from '@logto/schemas';
+import { ConnectorType } from '@logto/connector-core';
+import type { ConnectorResponse } from '@logto/core/src/connectors/types';
```

Comment:
The import path `@logto/core/src/connectors/types` seems to be a little strange since it contains `/src` part.

Can we move `ConnectorResponse` from `core` to `connector-core`?

Reply:
we can't. `ConnectorResponse` depends on `Connector` (the connector's DB schema defined in @logto/schemas and @logto/connector-core should not depends on @logto/schemas).
we can export type `ConnectorResponse` at src/index.ts, but it is not that general since its use case is pretty limited.

Reply:
Updated and only done rename in this PR.